boards/arm64/fvp_baser_aemv8r/fvp_baser_aemv8r.yaml

```diff
@@ -5,10 +5,9 @@ identifier: fvp_baser_aemv8r
 name: FVP Emulation FVP_BaseR_AEMv8R
 arch: arm64
 type: sim
+simulation: arm_fvp
```

Comment:
Will this config activate the test running on FVP? Since the FVP for cortex-R82 is not in CI/CD, will it cause a CI/CD failure?

Reply:
Or test could be disabled by removing `testing/default:true`? Just out of curiosity.

Reply:
yes, you are right, this is failing on CI. I'm extremely confused by CI as well so I don't have a real answer. @nashif @galak I think we should leave `simulation: arm_fvp` out until FVP is supported on CI/CD.